eventActions/tosReminderAction.js

```diff
@@ -0,0 +1,70 @@
+const Discord = require('discord.js');
+table = require('../databaseFiles/tosReminderTable');
+const config = require('../config.json');
+
+//Adds to database
+const addToDatabase = async function(user,joinTime,reminded=false){
+    table.sync().then(async ()=>{
+        await table.create({
+            user_id: user.id,
+            joinTime: joinTime,
+            reminded: reminded
+        }).catch(err =>{
+            console.error('Tosreminder error: ' + err )
+        })
+    })
+}
+
+//Removes user from Database
+const removeFromDatabase = async function(user){
+    await table.destroy({
+        where:{
+            user_id:user.id
+        }
+    }).catch(err =>{
+        console.error("Tos reminder error" + err)
+    })
+}
+
+//Reminds the user
+const tosRemind = async function(client){
+    messageEmbed = new Discord.MessageEmbed()
+     .setColor(config.colors.koaYellow)
+     .addField("Hey :wave: noticed you joined, but never got access to KOA.")
+     .addField(`Tap the check mark in https://discord.com/channels/382364344731828224/${config.channels.tos}/ in this message to have full access to all KOA channels. Enjoy your newfound powers :relieved:`)
```

Comment:
Check out `eventActions/tosActions.js` for an example of how to link a channel directly. It should look like `<#${config.channels.tos}>`.